language-configuration.json

```diff
@@ -0,0 +1,29 @@
+{
+  "comments": {
+    "lineComment": "#"
+  },
+  // symbols used as brackets
+  "brackets": [
+    ["{", "}"],
+    ["[", "]"],
+    ["(", ")"]
+  ],
+  // symbols that are auto closed when typing
+  "autoClosingPairs": [
+    { "open": "{", "close": "}" },
+    { "open": "[", "close": "]" },
+    { "open": "(", "close": ")" },
+    { "open": "'", "close": "'", "notIn": ["string", "comment"] },
```

Comment:
As of right now, the language does not have strings and there are currently no plans to add strings to the language; so this can safely be removed for now.